[2024-08-03T16:15:19.670+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: excel_to_oracle_dag.load_excel_to_oracle manual__2024-08-03T16:15:18.677791+00:00 [queued]>
[2024-08-03T16:15:19.681+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: excel_to_oracle_dag.load_excel_to_oracle manual__2024-08-03T16:15:18.677791+00:00 [queued]>
[2024-08-03T16:15:19.682+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-08-03T16:15:19.683+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2024-08-03T16:15:19.683+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-08-03T16:15:19.698+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): load_excel_to_oracle> on 2024-08-03 16:15:18.677791+00:00
[2024-08-03T16:15:19.704+0000] {standard_task_runner.py:55} INFO - Started process 8099 to run task
[2024-08-03T16:15:19.708+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'excel_to_oracle_dag', 'load_excel_to_oracle', 'manual__2024-08-03T16:15:18.677791+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/excel_to_oracle_dag.py', '--cfg-path', '/tmp/tmp18vivaiq']
[2024-08-03T16:15:19.711+0000] {standard_task_runner.py:83} INFO - Job 19: Subtask load_excel_to_oracle
[2024-08-03T16:15:19.770+0000] {task_command.py:376} INFO - Running <TaskInstance: excel_to_oracle_dag.load_excel_to_oracle manual__2024-08-03T16:15:18.677791+00:00 [running]> on host b6edf892b8ff
[2024-08-03T16:15:19.847+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=excel_to_oracle_dag
AIRFLOW_CTX_TASK_ID=load_excel_to_oracle
AIRFLOW_CTX_EXECUTION_DATE=2024-08-03T16:15:18.677791+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-03T16:15:18.677791+00:00
[2024-08-03T16:15:19.858+0000] {base.py:71} INFO - Using connection ID 'Connection bai test' for task execution.
[2024-08-03T16:15:19.859+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/excel_to_oracle_dag.py", line 33, in load_excel_to_oracle
    connection = cx_Oracle.connect(user=conn.login, password=conn.password, dsn=dsn)
cx_Oracle.DatabaseError: DPI-1047: Cannot locate a 64-bit Oracle Client library: "libclntsh.so: cannot open shared object file: No such file or directory". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
[2024-08-03T16:15:19.869+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=excel_to_oracle_dag, task_id=load_excel_to_oracle, execution_date=20240803T161518, start_date=20240803T161519, end_date=20240803T161519
[2024-08-03T16:15:19.882+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 19 for task load_excel_to_oracle (DPI-1047: Cannot locate a 64-bit Oracle Client library: "libclntsh.so: cannot open shared object file: No such file or directory". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help; 8099)
[2024-08-03T16:15:19.919+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2024-08-03T16:15:19.941+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
